Readme.md

```diff
@@ -6,26 +6,27 @@
 
 [![SensioLabsInsight](https://insight.sensiolabs.com/projects/0885c0ce-4b9f-4b89-aa9c-e8f9f7a315e0/big.png)](https://insight.sensiolabs.com/projects/0885c0ce-4b9f-4b89-aa9c-e8f9f7a315e0)
 
-Welcome to the *eZ Platform Kernel* (also known as *eZ Publish 6.x and higher kernel*). The kernel is the heart of *eZ Platform*, a modern
-CMS built on top of the Symfony (Full Stack) Framework. Containing an advanced Content Model, it allows you to structure any kind of content or content-like data in a future-proof Content Repository. It also aims to provide additional features for the MVC layer (Symfony) to increase your productivity.
+Welcome to the *eZ Platform Kernel*. It is the heart of *eZ Platform*, a modern CMS built on top of the Symfony (Full Stack) Framework.
+It contains an advanced Content Model, allowing you to structure any kind of content or content-like data in a future-proof Content Repository. The eZ Platform also aims to provide additional features for the MVC layer (Symfony) to increase your productivity.
 
 This code repository contains several layers of (and implementations of) APIs. *Kernel* refers to this being the core,
 as opposed to the *Full Stack* which has bundles, user interfaces, and installers, all configured to make a complete application.
 
-*In other words this repo is for core development; for fixes, features and documentation of the Platform kernel itself.*
+*This repository is for core development; for fixes, features and documentation of the eZ Platform Kernel itself.*
 
 
 ## What is eZ Platform?
 
-eZ Platform is a bottom-up rewrite of eZ Publish, so a conservative approach was taken on backwards compatibility. The first steps were taken by introducing it in 2012 as *eZ Publish Platform 5.0*. This bundled a *Legacy Stack* ("4.x") & a *Platform Stack* (formerly "5.x", or "new stack") together into one distribution. Starting in 2015 with *eZ Platform*, this is no longer the case. The *Platform* has matured to become its own modern, self-sufficient CMS/CMF, and it can be used to meet your needs without having to also deal with any *Legacy* elements.
+*eZ Platform* is modern, self-sufficient CMS/CMF, and it can be used to meet needs of developers and editorial teams. It has been in development since 2011.
+Current eZ Platform v2 is the 7th generation of eZ Publish, it is built on top of the Symfony 3.4LTS framework (Full Stack). 
```

Comment:
```suggestion
Current eZ Platform v2 is the 7th generation of the product (previously named eZ Publish). It is built on top of the Symfony 3.4LTS framework (Full Stack). 
```

Reply:
On `master` it will be Symfony 4/ Symfony5 LTS  ;-)